tutorials/evoked/30_eeg_erp.py

```diff
@@ -290,14 +295,14 @@
 ax.set(xlabel='Time (s)', ylabel='GFP (µV)', title='EEG')
 
 # %%
-# Analyzing regions of interest (ROIs): averaging across channels
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Analyzing regions of interest
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
please revert. ROIs is ambiguous

Reply:
Same reason here, the heading is too long. What about "Averaging across channels"?